vercel/.env.example

```
@@ -0,0 +1,12 @@
+# Facebook Auth Configuration
+FACEBOOK_APP_ID=your_facebook_app_id
+FACEBOOK_APP_SECRET=your_facebook_app_secret
+FACEBOOK_CALLBACK_URL=/auth/facebook/callback
+
+# Session Configuration
+SESSION_SECRET=your_session_secret_here
+SESSION_MAX_AGE=86400000 # 24 hours in milliseconds
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SESSION_MAX_AGE key should go before the SESSION_SECRET key

(UnorderedKey)